modules/attachments/README.md

```diff
@@ -0,0 +1,83 @@
+# SCC Profile Attachment
+
+This module creates SCC Profile's (https://cloud.ibm.com/docs/security-compliance?topic=security-compliance-build-custom-profiles&interface=ui). A profile attachment is an association between the set of resources that you want to evaluate and a profile that contains the specific controls that you want to evaluate.
```

Comment:
This module does not create `SCC Profile's` - it creates an `SCC Attachment` - please update